backend/src/services/rate-limit/rate-limit-service.ts

```diff
@@ -0,0 +1,44 @@
+import { BadRequestError } from "@app/lib/errors";
+
+import { TRateLimitDALFactory } from "./rate-limit-dal";
+import { TRateLimit, TRateLimitUpdateDTO } from "./rate-limit-types";
+
+type TRateLimitServiceFactoryDep = {
+  rateLimitDAL: TRateLimitDALFactory;
+};
+
+export type TRateLimitServiceFactory = ReturnType<typeof rateLimitServiceFactory>;
+
+export const rateLimitServiceFactory = ({ rateLimitDAL }: TRateLimitServiceFactoryDep) => {
+  const initRateLimits = async (): Promise<TRateLimit> => {
+    const rateLimit = await rateLimitDAL.create({});
```

Comment:
Keep a uuid of `0000-000` like admin cfg. Else it will cause if by accident there is two configuration. Same goes for find. Find by uuid of 00000-000... if that is the only document that should exist.